src/main/fc/rc_controls.h

```diff
@@ -59,7 +59,7 @@ typedef enum {
     STICK_CENTER = 0,
     THROTTLE_THRESHOLD,
     STICK_CENTER_ONCE
-} airmodeAndAntiWindupHandlingType_e;
```

Comment:
@DzikuVx When I had added STICK_CENTER_ONCE I found this bug. Wasn't the bug preventing the THROTTLE_THRESHOLD on MR from working?